### DISCUSSION TITLE
Is there a Discord server for Thorium? If not could someone (Alex, or another maintainer) make one?

### DISCUSSION THREAD

**Discussion by arjpar** [Alex313031/thorium#643] [Q&A]
I think it would be useful if there was a real time chat for the Thorium community.

**Comment by midzer** [comment]
A real time chat already exists and is linked on the website -> [Thorium IRC Channel (Web)](https://thorium.rocks/irc/)

**Reply by arjpar** [reply]
While IRC is great, many users (such as myself) are turned off to it due to the lack of native chat history backlog support. I know it is possible in IRC through using a bouncer like soju or ZNC, but it is somewhat tedious for (new) users to set up. Many open source projects use Discord or Element, services with many more users & native chat history backlog support. Perhaps a Discord (or Element) server could be synced/tethered to the IRC server. Just food for thought.
